### PR TITLE
v1.25.0

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

File: cli/compat/mod.rs [hunk 1]
```diff
@@ -17,7 +17,7 @@ pub use esm_resolver::NodeEsmResolver;
 
 // WARNING: Ensure this is the only deno_std version reference as this
 // is automatically updated by the version bump workflow.
-pub(crate) static STD_URL_STR: &str = "https://deno.land/std@0.151.0/";
+pub(crate) static STD_URL_STR: &str = "https://deno.land/std@0.153.0/";
```
Comment (dsherret) [inline cli/compat/mod.rs:20]: 152

**Review by dsherret — APPROVED** [review]
LGTM once deno_std version is fixed.

File: cli/compat/mod.rs (hunk 1 above)
Comment (dsherret) [reply cli/compat/mod.rs:20]: Oh, wait it is 153

**Comment by g2p** [comment]
cargo install deno fails (1.25.0, on Linux).
The deno_ffi build script breaks because tinycc isn't included in the crate source tarball.

```
CARGO_TARGET_DIR=/tmp/cargo-installhXwaLD cargo install -j1 deno
    Updating crates.io index
  Installing deno v1.25.0
   Compiling deno_ffi v0.53.0
error: failed to run custom build command for `deno_ffi v0.53.0`

Caused by:
  process didn't exit successfully: `/tmp/cargo-installhXwaLD/release/build/deno_ffi-4402470cd9da2f6a/build-script-build` (exit status: 101)
  --- stderr
  [/home/g2p/.cargo/registry/src/github.com-1ecc6299db9ec823/deno_ffi-0.53.0/build.rs:29] &tcc_src = "/home/g2p/.cargo/registry/src/github.com-1ecc6299db9ec823/deno_ffi-0.53.0/tinycc"
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', /home/g2p/.cargo/registry/src/github.com-1ecc6299db9ec823/deno_ffi-0.53.0/build.rs:34:37
  ```
  
Confirmed that https://crates.io/api/v1/crates/deno_ffi/0.53.0/download doesn't contain a tinycc folder.

**Pull request merged**